### PR TITLE
{vis}[foss/2022a] Ferret v7.6.0

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

* [x] Depends on https://github.com/easybuilders/easybuild-easyblocks/pull/3052

### CODE REVIEW

**Comment by lexming** [comment]
@boegelbot: please test @ generoso
EB_ARGS="--include-easyblocks-from-pr=3052"

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--include-easyblocks-from-pr=3052"

**Comment by lexming** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr=3052"

File: easybuild/easyconfigs/f/Ferret/Ferret-7.6.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    ('cURL', '7.83.0'),
+    ('ncurses', '6.3'),
+    ('libreadline', '8.1.2'),
+    ('tcsh', '6.24.01'),
+    ('Java', '11', '', SYSTEM),
+]
+
+# /bin/csh is required for installation by bin/make_executable_tar, bin/make_environment_tar
+# and also required by many installed scripts in bin/
+osdependencies = ['tcsh']
```
Comment (branfosj) [inline easybuild/easyconfigs/f/Ferret/Ferret-7.6.0-foss-2022a.eb:32]: `tcsh` is in the dependencies. Is this needed as an OS dep as well?

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/Ferret/Ferret-7.6.0-foss-2022a.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/f/Ferret/Ferret-7.6.0-foss-2022a.eb]: ah yeah, this is indeed redundant, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/19415/commits/360c1c3949fff5fec2298c3e8ba3f87d9ff0760f thanks

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/fd9217bceb70e0d6eb1e2dec1cfa0a31 for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @lexming!

**Pull request merged**